src/qt/locale/bitcoin_nl.ts

```diff
@@ -4421,75 +4421,75 @@ p, li { white-space: pre-wrap; }
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Stelt de serialisatie van onbewerkte transactie of blokhex in die wordt geretourneerd in niet-uitgebreide modus, non-segwit(0) or segwit(1) (default: %d)</translation>
     </message>
     <message>
         <source>This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit %s and cryptographic software written by Eric Young and UPnP software written by Thomas Bernard. Paper wallet art provided by Anacoluthia.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Dit product bevat software die is ontwikkeld door het OpenSSL-project voor gebruik in de OpenSSL Toolkit %s en cryptografische software die is geschreven door Eric Young en UPnP-software die is geschreven door Thomas Bernard. Papieren portemonnee-kunst geleverd door Anacoluthia.</translation>
     </message>
     <message>
         <source>Amaze</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verbazend</translation>
     </message>
     <message>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Standaard</translation>
     </message>
     <message>
         <source>Many generous</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Heel genereus</translation>
     </message>
     <message>
         <source>Minimum</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Minimum</translation>
     </message>
     <message>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Meer</translation>
     </message>
     <message>
         <source>Receive and display P2P network alerts (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">P2P-netwerkwaarschuwingen ontvangen en weergeven (default: %u)</translation>
     </message>
     <message>
         <source>Such expensive</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Zo duur</translation>
     </message>
     <message>
         <source>Wow</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Wow</translation>
     </message>
     <message>
         <source>-discardthreshold is set very high! This is the output amount that the wallet will discard (to fee) if it is smaller than this setting.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">-afgooidrempel is erg hoog! Dit is het uitvoerbedrag dat de portemonnee zal weggooien (tegen vergoeding) als het kleiner is dan deze instelling.</translation>
```

Comment:
- `-afgooidrempel` => `-discardthreshold` - should not be translated because this is the parameter name
- `uitvoerbedrag` => `wisselgeld` or `uitgaand bedrag` - "output" in this context is "outbound" rather than "export"
- `(tegen vergoeding)` => `(als transactiekosten)` - less confusing (we use both "vergoeding" and "transactiekosten" interchangeably, but here it is important to make clear that the user will lose the money.

Reply:
Changed `uitvoerbedrag` => `uitgaand bedrag`